docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
```

Comment:
I modelled the structure on the Kubernetes example, with groups in `example` and walkthrough in `docs`.
So for now, I'd like to keep it as is.

For discussion on the locations, see: https://github.com/giantswarm/inago/issues/219
For `docs` vs. `example` naming inconsistency, see: https://github.com/giantswarm/inago/issues/220